content/getting-started/joining-groups.md

```diff
@@ -1,34 +1,40 @@
 +++
 title = "Joining groups"
-tag = "additional"
+tag = "around"
 weight = 2
 +++
 
-### 1. Get on the network
+Joining public groups is straightforward: search for the group host's Urbit ID, and join. Some groups are private, so you'll need approval to join.
 
-First, you’ll need to get on Urbit.
+And still other groups are secret!  Which means you can't find them, and you'll need to be invited to join.
 
-Check out our [Getting Started guide](https://urbit.org/install) to find the best way for you.
+Let's walk through the process of finding and joining a public group.
 
-### 2. Open the Groups app
 
-Once you’re on the network, you’ll see and large tiled interface with a few applications installed: Talk, Terminal, Groups, and Bitcoin. Click on Groups.
+### 1. Open the Groups app
+
+Once you’re on the network, you’ll see Landscape, which is filled with large applications tiles: Talk, Terminal, and Groups. Click on Groups.
 
 ![](https://media.urbit.org/site/additional-guides/groups-1.png)
 
-### 3. Enter the group shortcode
+### 2. Enter the group shortcode
+
+To join a group, you'll need to know its host Urbit ID (e.g. ~bitbet-bolbel) or its full Urbit address, also called a shortcode.
+
+Check out some public groups in the [urbit.org groups directory](/ecosystem?type=groups)
 
-We’ll start by joining UF, the official group of the Urbit Foundation.
+As an example, we’ll start by joining UF, the official group of the Urbit Foundation.
 
 In the Groups app, click on "Find Groups" in the left column and enter `~halbex-palheb/uf-public`
 
 Then click on the "Join" button once the group appears.
 
 ![](https://media.urbit.org/site/additional-guides/groups-2.png)
 
-### 4. Join channels
 
-Now, you'll see the first default channel of the group.
+### 3. Join channels
+
+Give it a minute and you'll see the first default channel of the group.  For some larger groups with a large backlog, such as Urbit Community, this may require for than a minute.
```

Comment:
```suggestion
After a few moments the default channels will load. For groups with a large backlog, such as Urbit Community (~bitbet-bolbel/urbit-community), this may take a few minutes.
```